src/git.ts

```diff
@@ -789,7 +790,7 @@ export function handleGitDiff(actionData: any) {
 
     const ref1: string = actionData.params?.ref1;
     const ref2: string = actionData.params?.ref2;
-    const filePath: string = actionData.params?.filePath;
+    const filePath: string = actionData.params?.filePath || repo.rootUri.fsPath;
```

Comment:
Apparently git doesn't like absolute paths (?) 😓
`"."` does what we want.
```suggestion
    const filePath: string = actionData.params?.filePath || ".";
```

Reply:
inconsistency is fun